docs/guide/profiling-test-performance.md

```diff
@@ -0,0 +1,81 @@
+# Profiling Test Performance
+
+When you run Vitest it reports multiple time metrics of your tests:
+
+> ```bash
+> RUN  v2.1.1 /x/vitest/examples/profiling
+>
+> ✓ test/prime-number.test.ts (1) 4517ms
+>   ✓ generate prime number 4517ms
+>
+> Test Files  1 passed (1)
+>      Tests  1 passed (1)
+>   Start at  09:32:53
+>   Duration  4.80s (transform 44ms, setup 0ms, collect 35ms, tests 4.52s, environment 0ms, prepare 81ms)
+>   # Time metrics ^^
+> ```
+
+- Transform: How much time was spent transforming the files.
+- Setup: Time spent for running the [`globalSetup`](/config/#globalsetup) files.
+- Collect: Time spent for colleting all tests in the test files. This includes the time it took to import all file dependencies.
+- Tests: Time spent for actually running the test cases.
+- Environment: Time spent for setting up the test environment, for example JSDOM.
+- Prepare: ???
+
+## Test runner
+
+In cases where your test execution time is high, you can generate a profile of the test runner. Define `poolOptions.<pool>.execArgv` in your Vitest configuration:
+
+::: code-group
+```ts [Forks]
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    pool: 'forks',
+    poolOptions: {
+      forks: {
+        // TODO, requires https://github.com/nodejs/node/issues/55094
+        execArgv: ['--cpu-prof', '--cpu-prof-dir=test-runner-profile'],
+
+        // To generate a single profile
+        singleFork: true,
+      },
```

Comment:
There's now https://github.com/nodejs/node/issues/55094 but it might get closed as "works as intended" - let's hope not. 

If no changes in Node are introduced, we'll need to implement graceful exits in Vitest. Tinypool supports this already.

Reply:
Work-around added in https://github.com/vitest-dev/vitest/pull/6555/commits/cf10bf7129f784818f629bec47df1ccc5c466b1f